src/main/java/com/ethan/twfaith/powers/blessings/HellsFury.java

```diff
@@ -20,15 +20,16 @@ public void onHellsFuryTrigger(Player player, PlayerData player_data, ItemStack
         for (Player nearby : Bukkit.getOnlinePlayers()){
             PlayerData nearby_data = PlayerHashMap.playerDataHashMap.get(nearby.getUniqueId());
             if (!player.getWorld().equals(nearby.getWorld())){continue;}
-            if (player.getLocation().distance(nearby.getLocation()) >= 30 || !player_data.getLedBy().equals(nearby_data.getLedBy())){continue;}
-            if (nearby_data.isHellsFuryActive()){
+            if (nearby_data.isHellsFuryActive() && player_data.getLedBy().equals(nearby_data.getLedBy())){
+                continue;
```

Comment:
We are unable to use continue here because it skips to the next element, running the code from the top of the for loop. The code below continue will never be run.

Reply:
Without the continue statement, it looks like this would fix the bug.